packages/Uploader/src/Uploader.styles.js

```diff
@@ -0,0 +1,27 @@
+import { css } from "styled-components";
+import tokens from "@paprika/tokens";
+
+export const containerStyles = css`
+  display: inline-block;
+  position: relative;
+`;
+
+export const labelStyles = css`
+  cursor: pointer;
+  display: inline-block;
+  height: 100%;
+  position: absolute;
+  width: 100%;
+`;
+
+export const inputFileStyles = css`
+  border: 0;
+  height: 100%;
+  opacity: 0;
+  overflow: hidden;
+  position: absolute;
+  width: 100%;
+  &:focus + label {
```

Comment:
A bit odd that this bit of label styling is defined in the `inputFileStyles`.  Since the styling requires that `input` and `label` be siblings and are both descendants of the `container`, I'd suggest just combining it all into the `containerStyles`.